test/analysis/test_summary.py

```diff
@@ -2,8 +2,11 @@
 
 import pandas as pd
 
+import spras.config as config
 from spras.analysis.summary import summarize_networks
 
+config.init_from_file("config/config.yaml")
```

Comment:
This is an interesting question. Should we always init an config when testing even if the tests don't need it?